jsonschema-core.xml

```diff
@@ -800,49 +800,291 @@
+                    that use "$vocabulary" MUST explicitly list the Core vocabulary,
+                    which MUST have a value of true indicating that it is required.
+                </t>
+                <t>
+                    The behavior of a false value for this vocabulary (and only this
+                    vocabulary) is undefined, as is the behavior when "$vocabulary"
+                    is present but the Core vocabulary is not included.  However, it
+                    is RECOMMENDED that implementations detect these cases and raise
+                    an error when they occur.
+                </t>
+                <t>
+                    Meta-schemas that do not use "$vocabulary" MUST be considered to
+                    require the Core vocabulary just as if "$vocabulary" were used
+                    with only the Core vocabulary present with a value of true.
+                </t>
+                <t>
+                    The current URI for the Core vocabulary is:
+                    <eref target="https://json-schema.org/draft-08/vocabularies/core"/>.
+                </t>
+                <t>
+                    The current URI for the corresponding meta-schema is:
+                    <eref target="https://json-schema.org/draft-08/core"/>.
+                </t>
+            </section>
+            <section title="Example Meta-Schema With Vocabulary Declarations"
+                     anchor="example-meta-schema">
+                <figure>
+                    <preamble>
+                        This meta-schema explicitly declares both the Core and Applicator
+                        vocabularies, and combines their meta-schemas with an "allOf".
+                        It additionally restricts the usage of the Applicator vocabulary
+                        by forbidding the keyword prefixed with "unevaluated".  It also
+                        describes a keyword, "localKeyord", that is not part of either
+                        vocabulary.  Note that it is its own meta-schema,
+                        as it relies on both the Core vocabulary (as all schemas do)
+                        and the Applicator vocabulary (for "allOf"0.
+                    </preamble>
+                    <artwork>
+<![CDATA[
+{
+  "$schema": "https://json-schema.org/draft-08/core-app-example#",
+  "$id": "https://json-schema.org/draft-08/core-app-example",
+  "$recursiveAnchor": true,
+  "$vocabulary": {
+    "https://json-schema.org/draft-08/vocabularies/core": true,
+    "https://json-schema.org/draft-08/vocabularies/applicator": true
+  },
+  "allOf": [
+    {"$ref": "https://json-schema.org/draft-08/core"},
+    {"$ref": "https://json-schema.org/draft-08/applicator"}
+  ],
+  "patternProperties": {
+    "^unevaluated.*$": false
+  },
+  "properties": {
+    "$comment": "Not in vocabulary, but validated if used",
+    "localKeyword": {
+      "type": "string"
+    }
+  }
+}
+]]>
+                    </artwork>
```

Comment:
"artwork" =]

Reply:
?

Is this just a random amusement smiley or did I misspell artwork?  I'm staring at it and don't see a problem.

Reply:
Amusement =]